src/app/parcels/parcelsTableFilters.ts

```diff
@@ -29,6 +30,14 @@ export const postcodeSearch = (
     query: PostgrestFilterBuilder<Database["public"], any, any>,
     state: string
 ): PostgrestFilterBuilder<Database["public"], any, any> => {
+    if (state === "") {
+        return query;
+    }
+    if (nullPostcodeDisplay.toLowerCase().includes(state.toLowerCase())) {
+        return query.or(
+            `client_address_postcode.ilike.%${state}%, client_address_postcode.is.null`
+        );
+    }
```

Comment:
we need to change the postcode filter to search for null postcodes if "nfa" is searched for